bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -0,0 +1,97 @@
+# TapoControl Binding
+
+This binding adds support to control Tapo (Copyright © TP-Link Corporation Limited) Smart Home Devices from your local openHAB system.
+(Note: This is not an official Tapo/TP-link Binding!)
+
+## Supported Things
+
+The following Tapo-Devices are supported
+
+### P100 SmartPlug (WiFi)
+
+* Power On/Off
+* Wi-Fi signal (SignalStrenght)
+* On-Time (Time in seconds device is switched on)
+
+
+### L530_Series MultiColor SmartBulb (WiFi)
+
+* Light On/Off
+* Brightnes (Dimmer)  0-100 %
+* ColorTemperature (Number) 2500-6500 K
+* Color (Color)
+* Wi-Fi signal (SignalStrenght)
+* On-Time (Time in seconds device is switched on)
+
+## Prerequisites
+
+Before using Smart Plugs with openHAB the devices must be connected to the Wi-Fi network.
+This can be done using the Tapo provided mobile app.
+
+## Discovery
+
+Discovery is not supported at the moment. You need to know the IP-Adress of your device
+
+## Binding Configuration
+
+Binding needs your Tapo eMail and password to connect to the Tapo-Cloud.
+This is only used to create the handshake (cookie) to act with your devices.
```

Comment:
What's the reason to bind the credentials to the binding and not to a bridge?

Reply:
In my opninion is a bridge a device control other devices. But there's not such a device. So i decided to put them to the binding.

Reply:
The bridge doesn't need to be an actual device. It's sufficient to use the cloud to access the devices to implement it as a bridge.

If you implement the cloud access as a bridge, you can also use more than one cloud access on the same OH. This is more consistent with the OH concepts.

Reply:
From openhab documentation about Bridges i read
> A BridgeHandler handles the communication between the openHAB framework and a bridge (a device that acts as a gateway to enable the communication with other devices) represented by a Bridge instance.

So i thought it would be a binding configuration.
Is it necessary to change that or just a suggestion?

Reply:
It's better to change it to provide a consistent configuration approach for the users.

Reply:
Ok i'll try to change it soon